go/capture/driver/airbyte/connector.go

```diff
@@ -361,18 +372,18 @@ func (o *jsonOutput) Close() error {
 }
 
 type connectorStderr struct {
-	err bytes.Buffer
+	delegate io.Writer
```

Comment:
I was intending to forward stderr lines to the logs collection in #243 but I was holding off there because I wasn't sure how to parse the stderr output. I now have a bit more conviction on how to handle it there, so should be able to add that into the PR. I don't think there's any need to change anything in _this_ PR, but just figured I'd mention it since I'd be trampling over this code assuming we merge this first.

Reply:
Ack